saio/bin/start_proxyfs_and_swift

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -x
```

Comment:
Not sure that you wanted to check this in. Kinda nice to keep this script quieter imho.